codescan/schema.go

```diff
@@ -216,7 +216,17 @@ func (s *schemaBuilder) buildFromDecl(_ *entityDecl, schema *spec.Schema) error
 				if ti.IsBuiltin() {
 					break
 				}
+
 				if ti.IsType() {
+					if _, ok := ti.Type.(interface{ TypeParams() *types.TypeParamList }); ok {
+						if underlyingStruct, ok := o.Type().Underlying().(*types.Struct); ok {
```

Comment:
Probably should also check that the len(type params) is > 0, otherwise this will be triggered for any type alias

Also, would be nice to make these `if`'s cleaner

Reply:
nvm, outdated